src/mscorlib/shared/System/IO/FileStream.Win32.cs

```diff
@@ -3,14 +3,29 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.Win32.SafeHandles;
-using System.Runtime.InteropServices;
 
 namespace System.IO
 {
     public partial class FileStream : Stream
     {
+        private static bool IsFromAppAvailable()
+        {
+            SafeLibraryHandle handle = Interop.Kernel32.LoadLibraryExW(Interop.Libraries.FileApiInterop, IntPtr.Zero, 0);
```

Comment:
Good catch, @jkotas . Yes, that's very important. Where is FileApiInterop.dll expected to be deployed?